code/modules/surgery/bodyparts/species_parts/lizard_bodyparts.dm

```diff
@@ -10,6 +10,9 @@
 	is_dimorphic = TRUE
 	wing_types = list(/obj/item/organ/external/wings/functional/dragon)
 
+/obj/item/bodypart/chest/lizard/get_butt_sprite()
+	return BUTT_SPRITE_LIZARD
```

Comment:
this seems out of place in context of the above

Reply:
as discussed on discord, sprite for lizards is scaly, so wouldn't make sense if just owning a lizard tail showed a scaly sprite